src/bio/bio_context.c

```diff
@@ -332,11 +360,15 @@ bio_blob_delete(uuid_t uuid, struct bio_xs_context *xs_ctxt)
 	blob_wait_completion(xs_ctxt, ba);
 	rc = ba->bca_rc;
 
-	if (rc != 0)
+	bmem.bmem_bs = bbs;
+	bmem.bmem_rc = rc;
+
+	if (rc != 0) {
+		spdk_thread_send_msg(owner_thread(bbs), bio_media_error, &bmem);
```

Comment:
I think it's not necessary to log the blobstore MD errors. Logging only the blob I/O & unmap errors should be fine.